tooling/template-base/renderer/_default.page.server.tsx

```diff
@@ -26,39 +23,25 @@ export const passToClient = [
   'authToken'
 ];
 
-async function render(pageContext: PageContext) {
-  const { pageHtml } = pageContext;
+export { render };
+
+type RenderFn = (pageContext: PageContext) => Promise<{
+  documentHtml: ReturnType<typeof escapeInject>;
+  pageContext: Record<string, any>;
+}>;
+const render: RenderFn = async pageContext => {
```

Comment:
Got TS error `Return type of exported function has or is using name 'TemplateWrapped' from external module "/Users/lu.jiang/Documents/_src/helium/tooling/template-base/node_modules/vite-plugin-ssr/dist/cjs/node/html/renderHtml" but cannot be named.ts(4058)`.

Used TS utility to explicitly type the render function instead.